packages/react-components/src/components/menu/menu.css

```diff
@@ -0,0 +1,25 @@
+.iot-context-menu-shadow {
+  box-shadow: rgba(0, 7, 22, 0.1) 0px 4px 20px 1px;
```

Comment:
is there a design token color we could use ? probably have to move to js to use the design token

Reply:
unfortunately no, cloudscape doesn't expose tokens for drop shadows